core/src/main/scala/cats/Eval.scala

```diff
@@ -350,7 +350,7 @@ trait EvalMonoid[A] extends Monoid[Eval[A]] with EvalSemigroup[A] {
 trait EvalGroup[A] extends Group[Eval[A]] with EvalMonoid[A] {
   implicit def algebra: Group[A]
   def inverse(lx: Eval[A]): Eval[A] =
-    lx.map(_.inverse)
+    lx.map(_.inverse())
```

Comment:
It isn't needed, but the definition of that function in `cats.syntax.GroupOps` is:

```scala
def inverse(): A = macro Ops.unop[A]
```

Not sure why that's the definition, but when you've got a parens pair like that in your `def`, then you have tools like IntelliJ IDEA complaining about you not using the parens at the call site. Which I personally find cool because those parens usually indicate side-effects. Maybe we should change the `def inverse()` definition?